PWGMM/Mult/Tasks/dndeta-hi.cxx

```diff
@@ -234,82 +277,102 @@ struct MultiplicityCounter {
-          registry.fill(HIST("hrecdndeta"), Double_t(kINEL), 50., z, ttrack.mcParticle_as<Particles>().eta());
+          // registry.fill(HIST("hrecdndeta"), Double_t(kDATA),
+          // Double_t(kMBAND), 50., z, ttrack.mcParticle_as<Particles>().eta());
+          registry.fill(HIST("hrecdndeta"), Double_t(kINEL), Double_t(kMBAND),
+                        50., z, ttrack.mcParticle_as<Particles>().eta());
         } else {
           // when secondary
         }
       }
       for (auto& track : tracks) {
-        if (std::find(usedTracksIds.begin(), usedTracksIds.end(), track.globalIndex()) != usedTracksIds.end()) {
+        if (std::find(usedTracksIds.begin(), usedTracksIds.end(),
+                      track.globalIndex()) != usedTracksIds.end()) {
           continue;
         }
         if (track.has_mcParticle()) {
-          // registry.fill(HIST("hrecdndeta"), Double_t(kDATA), Double_t(kMBAND), 50., z, track.mcParticle_as<Particles>().eta());
-          registry.fill(HIST("hrecdndeta"), Double_t(kINEL), 50., z, track.mcParticle_as<Particles>().eta());
-          // registry.fill(HIST("hrecdndeta"), Double_t(kINEL), Double_t(kMBAND), 50., z, 2);
+          // registry.fill(HIST("hrecdndeta"), Double_t(kDATA),
+          // Double_t(kMBAND), 50., z, track.mcParticle_as<Particles>().eta());
+          registry.fill(HIST("hrecdndeta"), Double_t(kINEL), Double_t(kMBAND),
+                        50., z, track.mcParticle_as<Particles>().eta());
+          // registry.fill(HIST("hrecdndeta"), Double_t(kINEL),
+          // Double_t(kMBAND), 50., z, 2);
         } else {
           // when secondary
         }
       }
     }
   }
-  PROCESS_SWITCH(MultiplicityCounter, processMCCounting, "MC Count tracks", false);
+  PROCESS_SWITCH(MultiplicityCounter, processMCCounting, "MC Count tracks",
+                 false);
 
-  Partition<aod::MFTTracks> sample = (aod::fwdtrack::eta < -2.8f) && (aod::fwdtrack::eta > -3.2f);
-  expressions::Filter atrackFilterForward = (aod::fwdtrack::bestCollisionId >= 0) &&
-                                            (aod::fwdtrack::etas < -2.0f) &&
-                                            (aod::fwdtrack::etas > -3.9f) &&
-                                            (nabs(aod::fwdtrack::bestDCAXY) <= 2.f);
-  void processForwardCounting(soa::Join<aod::Collisions, aod::EvSels>::iterator const& collision, aod::MFTTracks const& tracks, soa::SmallGroups<soa::Join<aod::AmbiguousMFTTracks, aod::BestCollisionsFwd>> const& atracks)
-  {
-    if (!useEvSel || (useEvSel && collision.sel8())) {
-      usedTracksIds.clear();
-      auto z = collision.posZ();
-      for (auto& track : atracks) {
-        usedTracksIds.emplace_back(track.mfttrackId());
-        registry.fill(HIST("hrecdndeta"), 1, 1, 50., z, track.etas());
-      }
-      for (auto& track : tracks) {
-        if (std::find(usedTracksIds.begin(), usedTracksIds.end(), track.globalIndex()) != usedTracksIds.end()) {
-          continue;
+  /*
```

Comment:
Here and elsewhere - please do not leave large commented parts of code in the files. The commit history is stored and this code would no be lost.